nncf/experimental/torch/sparsity/movement/algo.py

```diff
@@ -116,6 +115,23 @@ def _build_controller(self, model: NNCFNetwork) -> PTCompressionAlgorithmControl
         return MovementSparsityController(model, self._sparsified_module_info, self.config)
 
 
+
+MODEL_FAMILIES = ['bert', 'wav2vec2', 'swin']
```

Comment:
@vuiseng9 @yujiepan-work I temporarily left checking for supported model family, since this PR introduce support for just 3 models.